### ISSUE TITLE
Build failed, do you want to continue? (but code still runs)

### ISSUE THREAD

**Issue by benjs2000** [rage/tmc-vscode#275]
Hi,

I'm having an issue where I get this error every time I try to run my code. I reset my PC and wasn't getting this error before. If I hit "proceed" it still runs but I have to click that every time I run the code.
I have installed the JDK 11 as suggested for the course (Java Programming 1)

Any suggestions?
Thanks!

![Capture](https://user-images.githubusercontent.com/61516436/77746384-c54c7300-7014-11ea-9a22-26989e51ff24.PNG)

**Comment by jesseanttila-cai** [comment]
Hi,

This seems to be an issue with [another](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug) extension, although I was able to reproduce the issue as well. I was also able to at least temporarily fix it for myself by selecting the "Fix..." option and choosing to clear the workspace cache. After doing that, builds seem to go through normally without any problems.

**Comment by benjs2000** [comment]
Hi,

Unfortunately clearing the workspace cache did not resolve it. I think it might be something to do with these errors? Do you know what the problem is?I installed the OpenJDK11 as suggested for the MOOC course... Thanks!

**Comment by jesseanttila-cai** [comment]
In that case you can select the option from the "Fix..." menu that brings up the error log, and paste it here. However, given the fact that the TMC extension is only responsible for running the tests for exercises, it is possible that there may not be anything we can do about this particular issue, as it involves running the exercise code through another extension.

If that ends up being the case, you'll have to open an issue on the [vscode-java](https://github.com/redhat-developer/vscode-java/issues) extension repository, as the problem would then originate from their side.

**Comment by benjs2000** [comment]
Hi again,

I really appreciate your help. I get that it might not be an issue specific to the TMC plugin. I do however not get problems when running code not within the MOOC exercises, if that makes a difference.
Below is both the problems expressed in the console and also the log file you asked for. Many thanks

![Capture](https://user-images.githubusercontent.com/61516436/77915617-b1607580-728f-11ea-99de-d78f07358885.PNG)

!SESSION 2020-03-29 12:09:27.488 -----------------------------------------------
eclipse.buildId=unknown
java.version=11.0.6
java.vendor=AdoptOpenJDK
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_GB
Command-line arguments:  -data c:\Users\Ben\AppData\Roaming\Code\User\workspaceStorage\c49807c10603be5ab32d880db5dc0acb\redhat.java\jdt_ws

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:28.963
!MESSAGE class org.eclipse.jdt.ls.core.internal.JavaLanguageServerPlugin is started

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.125
!MESSAGE Main thread is waiting

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.143
!MESSAGE >> initialize

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.149
!MESSAGE Initializing Java Language Server 0.52.0.202003051408

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.170
!MESSAGE Static Commands: []

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.170
!MESSAGE Non-Static Commands: [java.edit.organizeImports, java.project.getSettings, java.project.refreshDiagnostics, java.project.updateSourceAttachment, java.project.removeFromSourcePath, java.project.listSourcePaths, java.project.isTestFile, java.project.getClasspaths, java.project.resolveSourceAttachment, java.project.addToSourcePath]

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.171
!MESSAGE Static Commands: []

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.171
!MESSAGE Non-Static Commands: [java.maven.initializeSearcher, java.maven.searchArtifact, java.maven.addDependency, java.maven.controlContext]

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.171
!MESSAGE Static Commands: []

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.171
!MESSAGE Non-Static Commands: [vscode.java.test.search.codelens, vscode.java.test.search.items.all, vscode.java.test.search.location, vscode.java.test.search.items, vscode.java.test.get.testpath, vscode.java.test.junit.argument]

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.172
!MESSAGE Static Commands: []

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.172
!MESSAGE Non-Static Commands: [java.project.refreshLib, java.project.list, java.getPackageData, java.resolvePath]

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.172
!MESSAGE Static Commands: []

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.172
!MESSAGE Non-Static Commands: [vscode.java.checkProjectSettings, vscode.java.isOnClasspath, vscode.java.fetchUsageData, vscode.java.validateLaunchConfig, vscode.java.resolveMainMethod, vscode.java.resolveClasspath, vscode.java.resolveBuildFiles, vscode.java.resolveMainClass, vscode.java.updateDebugSettings, vscode.java.buildWorkspace, vscode.java.startDebugSession, vscode.java.inferLaunchCommandLength, vscode.java.resolveElementAtSelection, vscode.java.resolveJavaExecutable]

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.173
!MESSAGE Static Commands: []

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.173
!MESSAGE Non-Static Commands: [java.intellicode.completion, java.intellicode.completion.resolve, java.intellicode.enable]

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.186
!MESSAGE >> initialized

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.208
!MESSAGE Creating the Java project jdt.ls-java-project

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:29.308
!MESSAGE Finished creating the Java project jdt.ls-java-project

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:30.015
!MESSAGE Importing Maven project(s)

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:30.136
!MESSAGE Projects size:103

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:43.892
!MESSAGE Workspace initialized in 14717ms

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:43.931
!MESSAGE >> initialization job finished

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:43.937
!MESSAGE >> workspace/didChangeConfiguration

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:43.952
!MESSAGE >> New configuration: {java={home=C:\Program Files\AdoptOpenJDK\jdk-11.0.6.10-hotspot\, jdt={ls={vmargs=-XX:+UseParallelGC -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xmx1G -Xms100m}}, errors={incompleteClasspath={severity=warning}}, configuration={checkProjectSettingsExclusions=true, updateBuildConfiguration=interactive, maven={userSettings=null}, runtimes=[]}, trace={server=off}, import={maven={enabled=true}, gradle={enabled=true, wrapper={enabled=true}, offline={enabled=false}, version=null, arguments=null, jvmArguments=null, home=null}, exclusions=[**/node_modules/**, **/.metadata/**, **/archetype-resources/**, **/META-INF/maven/**]}, maven={downloadSources=false, updateSnapshots=false}, referencesCodeLens={enabled=false}, signatureHelp={enabled=false}, implementationsCodeLens={enabled=false}, format={enabled=true, settings={url=null, profile=null}, comments={enabled=true}, onType={enabled=true}}, saveActions={organizeImports=false}, project={referencedLibraries=[lib/**/*.jar]}, contentProvider={preferred=null}, autobuild={enabled=true}, maxConcurrentBuilds=1.0, completion={maxResults=50.0, enabled=true, overwrite=true, guessMethodArguments=false, favoriteStaticMembers=[org.junit.Assert.*, org.junit.Assume.*, org.junit.jupiter.api.Assertions.*, org.junit.jupiter.api.Assumptions.*, org.junit.jupiter.api.DynamicContainer.*, org.junit.jupiter.api.DynamicTest.*, org.mockito.Mockito.*, org.mockito.ArgumentMatchers.*, org.mockito.Answers.*], filteredTypes=[java.awt.*, com.sun.*], importOrder=[java, javax, com, org]}, foldingRange={enabled=true}, progressReports={enabled=true}, codeGeneration={hashCodeEquals={useJava7Objects=false, useInstanceof=false}, useBlocks=false, generateComments=false, toString={template=${object.className} [${member.name()}=${member.value}, ${otherMembers}], codeStyle=STRING_CONCATENATION, skipNullValues=false, listArrayContents=true, limitElements=0.0}}, selectionRange={enabled=true}, showBuildStatusOnStart={enabled=false}, debug={logLevel=warn, settings={showHex=false, showStaticVariables=false, showQualifiedNames=false, showLogicalStructure=true, showToString=true, maxStringLength=0.0, numericPrecision=0.0, hotCodeReplace=manual, enableRunDebugCodeLens=true, forceBuildBeforeLaunch=true, console=integratedTerminal}}, dependency={showMembers=false, syncWithFolderExplorer=true, autoRefresh=true, refreshDelay=2000.0, packagePresentation=flat}, help={firstView=gettingStarted}, test={report={showAfterExecution=onFailure, position=sideView}, editor={enableShortcuts=true}, log={level=info}, message={hintForDeprecatedConfig=true, hintForSetingDefaultConfig=true}, defaultConfig=, config={}}}}

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:43.954
!MESSAGE >> document/didOpen

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:44.256
!MESSAGE >> document/didOpen

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:44.274
!MESSAGE >> workspace/executeCommand java.intellicode.enable

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:44.280
!MESSAGE >> workspace/executeCommand vscode.java.updateDebugSettings

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:44.282
!MESSAGE >> document/documentSymbol

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:44.284
!MESSAGE >> document/foldingRange

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:44.286
!MESSAGE >> workspace/executeCommand vscode.java.resolveMainMethod

!ENTRY java-debug 1 0 2020-03-29 12:09:44.295
!MESSAGE Starting com.microsoft.java.debug.plugin

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:44.439
!MESSAGE >> java/buildWorkspace (incremental)

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:44.512
!MESSAGE >> workspace/executeCommand vscode.java.test.get.testpath

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:44.690
!MESSAGE >> workspace/executeCommand vscode.java.test.search.codelens

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:44.691
!MESSAGE >> workspace/executeCommand vscode.java.resolveMainMethod

!ENTRY org.eclipse.jdt.ls.core 4 0 2020-03-29 12:09:49.091
!MESSAGE Error occured while building workspace. Details: 
 message: lista cannot be resolved; code: 570425394; resource: C:/Users/Ben/AppData/Roaming/Code/User/globalStorage/moocfi.test-my-code/tmcdata/TMC workspace/Exercises/mooc/mooc-java-programming-i/part03-Part03_01.ThirdElement/src/main/java/ThirdElement.java; line: 20
 message: luvut cannot be resolved; code: 570425394; resource: C:/Users/Ben/AppData/Roaming/Code/User/globalStorage/moocfi.test-my-code/tmcdata/TMC workspace/Exercises/mooc/mooc-java-programming-i/part03-Part03_02.SecondPlusThird/src/main/java/SecondPlusThird.java; line: 20

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:49.167
!MESSAGE Reconciled 0. Took 80 ms

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:49.702
!MESSAGE begin problem for /AdaLovelace.java

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:49.702
!MESSAGE 0 problems reported for /AdaLovelace.java

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:49.707
!MESSAGE begin problem for /Sandbox.java

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:49.708
!MESSAGE 0 problems reported for /Sandbox.java

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:49.708
!MESSAGE Validated 2. Took 85 ms

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:50.704
!MESSAGE >> build jobs finished

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:51.746
!MESSAGE >> registerFeature 'workspace/didChangeWatchedFiles'

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:51.748
!MESSAGE >> registerFeature 'workspace/didChangeWatchedFiles'

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:09:51.749
!MESSAGE >> watchers registered

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:04.546
!MESSAGE >> document/documentSymbol

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:04.548
!MESSAGE >> document/documentSymbol

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:04.548
!MESSAGE User abort while collecting the document symbols.

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:04.548
!MESSAGE >> document/documentSymbol

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:04.549
!MESSAGE User abort while collecting the document symbols.

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:04.626
!MESSAGE >> document/documentSymbol

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:04.627
!MESSAGE >> document/documentSymbol

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:04.627
!MESSAGE >> document/documentSymbol

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:04.628
!MESSAGE User abort while collecting the document symbols.

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:04.628
!MESSAGE User abort while collecting the document symbols.

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:12.930
!MESSAGE >> document/didOpen

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:12.936
!MESSAGE >> workspace/executeCommand java.resolvePath

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:12.936
!MESSAGE >> document/documentSymbol

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:12.945
!MESSAGE >> workspace/executeCommand java.project.list

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:12.948
!MESSAGE >> workspace/executeCommand java.getPackageData

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:12.956
!MESSAGE >> workspace/executeCommand java.getPackageData

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:12.957
!MESSAGE >> workspace/executeCommand java.getPackageData

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:12.957
!MESSAGE >> workspace/executeCommand java.getPackageData

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:12.957
!MESSAGE >> workspace/executeCommand java.getPackageData

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:12.960
!MESSAGE >> workspace/executeCommand java.getPackageData

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:12.963
!MESSAGE >> workspace/executeCommand java.getPackageData

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:13.150
!MESSAGE >> document/foldingRange

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:13.172
!MESSAGE >> workspace/executeCommand vscode.java.test.search.codelens

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:13.172
!MESSAGE >> workspace/executeCommand vscode.java.resolveMainMethod

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:13.348
!MESSAGE Reconciled 0. Took 12 ms

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:13.807
!MESSAGE begin problem for /AdaLovelace.java

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:13.807
!MESSAGE 0 problems reported for /AdaLovelace.java

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:13.811
!MESSAGE begin problem for /Sandbox.java

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:13.811
!MESSAGE 0 problems reported for /Sandbox.java

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:13.815
!MESSAGE begin problem for /OnceUponATime.java

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:13.815
!MESSAGE 0 problems reported for /OnceUponATime.java

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:13.816
!MESSAGE Validated 3. Took 17 ms

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:14.749
!MESSAGE >> document/didClose

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:14.752
!MESSAGE >> workspace/executeCommand java.resolvePath

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:14.752
!MESSAGE >> document/documentSymbol

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:14.754
!MESSAGE >> workspace/executeCommand java.getPackageData

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:14.757
!MESSAGE >> workspace/executeCommand java.getPackageData

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:14.757
!MESSAGE >> workspace/executeCommand java.getPackageData

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:14.758
!MESSAGE >> workspace/executeCommand java.getPackageData

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:14.758
!MESSAGE >> workspace/executeCommand java.getPackageData

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:14.759
!MESSAGE >> workspace/executeCommand java.getPackageData

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:14.762
!MESSAGE >> workspace/executeCommand java.getPackageData

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:14.907
!MESSAGE >> document/foldingRange

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:14.957
!MESSAGE >> workspace/executeCommand vscode.java.test.search.codelens

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:10:14.958
!MESSAGE >> workspace/executeCommand vscode.java.resolveMainMethod

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:14:34.451
!MESSAGE >> shutdown

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:14:34.790
!MESSAGE >> exit

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:14:34.790
!MESSAGE Shutdown received... waking up main thread

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-29 12:14:34.794
!MESSAGE class org.eclipse.jdt.ls.core.internal.JavaLanguageServerPlugin is stopping:
!SESSION 2020-03-30 14:04:08.246 -----------------------------------------------
eclipse.buildId=unknown
java.version=11.0.6
java.vendor=AdoptOpenJDK
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_GB
Command-line arguments:  -data c:\Users\Ben\AppData\Roaming\Code\User\workspaceStorage\c49807c10603be5ab32d880db5dc0acb\redhat.java\jdt_ws

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:11.972
!MESSAGE class org.eclipse.jdt.ls.core.internal.JavaLanguageServerPlugin is started

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:12.126
!MESSAGE Main thread is waiting

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:12.148
!MESSAGE >> initialize

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:12.149
!MESSAGE Initializing Java Language Server 0.52.0.202003051408

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:12.192
!MESSAGE Static Commands: []

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:12.192
!MESSAGE Non-Static Commands: [java.edit.organizeImports, java.project.getSettings, java.project.refreshDiagnostics, java.project.updateSourceAttachment, java.project.removeFromSourcePath, java.project.listSourcePaths, java.project.isTestFile, java.project.getClasspaths, java.project.resolveSourceAttachment, java.project.addToSourcePath]

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:12.193
!MESSAGE Static Commands: []

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:12.193
!MESSAGE Non-Static Commands: [java.maven.initializeSearcher, java.maven.searchArtifact, java.maven.addDependency, java.maven.controlContext]

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:12.194
!MESSAGE Static Commands: []

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:12.194
!MESSAGE Non-Static Commands: [vscode.java.test.search.codelens, vscode.java.test.search.items.all, vscode.java.test.search.location, vscode.java.test.search.items, vscode.java.test.get.testpath, vscode.java.test.junit.argument]

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:12.195
!MESSAGE Static Commands: []

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:12.195
!MESSAGE Non-Static Commands: [java.project.refreshLib, java.project.list, java.getPackageData, java.resolvePath]

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:12.196
!MESSAGE Static Commands: []

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:12.196
!MESSAGE Non-Static Commands: [vscode.java.checkProjectSettings, vscode.java.isOnClasspath, vscode.java.fetchUsageData, vscode.java.validateLaunchConfig, vscode.java.resolveMainMethod, vscode.java.resolveClasspath, vscode.java.resolveBuildFiles, vscode.java.resolveMainClass, vscode.java.updateDebugSettings, vscode.java.buildWorkspace, vscode.java.startDebugSession, vscode.java.inferLaunchCommandLength, vscode.java.resolveElementAtSelection, vscode.java.resolveJavaExecutable]

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:12.197
!MESSAGE Static Commands: []

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:12.197
!MESSAGE Non-Static Commands: [java.intellicode.completion, java.intellicode.completion.resolve, java.intellicode.enable]

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:12.215
!MESSAGE >> initialized

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:13.248
!MESSAGE Importing Maven project(s)

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:14.060
!MESSAGE Workspace initialized in 1860ms

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:14.120
!MESSAGE >> initialization job finished

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:14.125
!MESSAGE >> workspace/didChangeConfiguration

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:14.140
!MESSAGE >> New configuration: {java={home=C:\Program Files\AdoptOpenJDK\jdk-11.0.6.10-hotspot\, jdt={ls={vmargs=-XX:+UseParallelGC -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Dsun.zip.disableMemoryMapping=true -Xmx1G -Xms100m}}, errors={incompleteClasspath={severity=warning}}, configuration={checkProjectSettingsExclusions=true, updateBuildConfiguration=interactive, maven={userSettings=null}, runtimes=[]}, trace={server=off}, import={maven={enabled=true}, gradle={enabled=true, wrapper={enabled=true}, offline={enabled=false}, version=null, arguments=null, jvmArguments=null, home=null}, exclusions=[**/node_modules/**, **/.metadata/**, **/archetype-resources/**, **/META-INF/maven/**]}, maven={downloadSources=false, updateSnapshots=false}, referencesCodeLens={enabled=false}, signatureHelp={enabled=false}, implementationsCodeLens={enabled=false}, format={enabled=true, settings={url=null, profile=null}, comments={enabled=true}, onType={enabled=true}}, saveActions={organizeImports=false}, project={referencedLibraries=[lib/**/*.jar]}, contentProvider={preferred=null}, autobuild={enabled=true}, maxConcurrentBuilds=1.0, completion={maxResults=50.0, enabled=true, overwrite=true, guessMethodArguments=false, favoriteStaticMembers=[org.junit.Assert.*, org.junit.Assume.*, org.junit.jupiter.api.Assertions.*, org.junit.jupiter.api.Assumptions.*, org.junit.jupiter.api.DynamicContainer.*, org.junit.jupiter.api.DynamicTest.*, org.mockito.Mockito.*, org.mockito.ArgumentMatchers.*, org.mockito.Answers.*], filteredTypes=[java.awt.*, com.sun.*], importOrder=[java, javax, com, org]}, foldingRange={enabled=true}, progressReports={enabled=true}, codeGeneration={hashCodeEquals={useJava7Objects=false, useInstanceof=false}, useBlocks=false, generateComments=false, toString={template=${object.className} [${member.name()}=${member.value}, ${otherMembers}], codeStyle=STRING_CONCATENATION, skipNullValues=false, listArrayContents=true, limitElements=0.0}}, selectionRange={enabled=true}, showBuildStatusOnStart={enabled=false}, debug={logLevel=warn, settings={showHex=false, showStaticVariables=false, showQualifiedNames=false, showLogicalStructure=true, showToString=true, maxStringLength=0.0, numericPrecision=0.0, hotCodeReplace=manual, enableRunDebugCodeLens=true, forceBuildBeforeLaunch=true, console=integratedTerminal}}, dependency={showMembers=false, syncWithFolderExplorer=true, autoRefresh=true, refreshDelay=2000.0, packagePresentation=flat}, help={firstView=gettingStarted}, test={report={showAfterExecution=onFailure, position=sideView}, editor={enableShortcuts=true}, log={level=info}, message={hintForDeprecatedConfig=true, hintForSetingDefaultConfig=true}, defaultConfig=, config={}}}}

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:14.141
!MESSAGE >> document/didOpen

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:14.268
!MESSAGE >> workspace/executeCommand java.intellicode.enable

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:14.273
!MESSAGE >> document/documentSymbol

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:14.662
!MESSAGE >> workspace/executeCommand vscode.java.resolveMainMethod

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:14.663
!MESSAGE >> document/foldingRange

!ENTRY java-debug 1 0 2020-03-30 14:04:14.686
!MESSAGE Starting com.microsoft.java.debug.plugin

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:14.689
!MESSAGE >> workspace/executeCommand vscode.java.test.get.testpath

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:14.919
!MESSAGE >> workspace/executeCommand vscode.java.test.search.codelens

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:14.920
!MESSAGE >> workspace/executeCommand vscode.java.resolveMainMethod

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:14.987
!MESSAGE Reconciled 0. Took 279 ms

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:15.516
!MESSAGE begin problem for /AdaLovelace.java

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:15.517
!MESSAGE 0 problems reported for /AdaLovelace.java

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:15.519
!MESSAGE Validated 1. Took 128 ms

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:16.899
!MESSAGE >> build jobs finished

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:17.168
!MESSAGE >> workspace/executeCommand vscode.java.updateDebugSettings

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:17.175
!MESSAGE >> java/buildWorkspace (incremental)

!ENTRY org.eclipse.jdt.ls.core 4 0 2020-03-30 14:04:17.189
!MESSAGE Error occured while building workspace. Details: 
 message: lista cannot be resolved; code: 570425394; resource: C:/Users/Ben/AppData/Roaming/Code/User/globalStorage/moocfi.test-my-code/tmcdata/TMC workspace/Exercises/mooc/mooc-java-programming-i/part03-Part03_01.ThirdElement/src/main/java/ThirdElement.java; line: 20
 message: luvut cannot be resolved; code: 570425394; resource: C:/Users/Ben/AppData/Roaming/Code/User/globalStorage/moocfi.test-my-code/tmcdata/TMC workspace/Exercises/mooc/mooc-java-programming-i/part03-Part03_02.SecondPlusThird/src/main/java/SecondPlusThird.java; line: 20

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:18.686
!MESSAGE >> registerFeature 'workspace/didChangeWatchedFiles'

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:18.688
!MESSAGE >> registerFeature 'workspace/didChangeWatchedFiles'

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:18.690
!MESSAGE >> watchers registered

!ENTRY org.eclipse.jdt.ls.core 1 0 2020-03-30 14:04:55.590
!MESSAGE >> workspace/executeCommand vscode.java.resolveBuildFiles

**Comment by jesseanttila-cai** [comment]
This error that appears in the log seems to suggest that the build is failing because all exercises are being built at the same time, and some of them are not in a compilable state.
``` !MESSAGE Error occured while building workspace. Details:
message: lista cannot be resolved; code: 570425394; resource: C:/Users/Ben/AppData/Roaming/Code/User/globalStorage/moocfi.test-my-code/tmcdata/TMC workspace/Exercises/mooc/mooc-java-programming-i/part03-Part03_01.ThirdElement/src/main/java/ThirdElement.java; line: 20
message: luvut cannot be resolved; code: 570425394; resource: C:/Users/Ben/AppData/Roaming/Code/User/globalStorage/moocfi.test-my-code/tmcdata/TMC workspace/Exercises/mooc/mooc-java-programming-i/part03-Part03_02.SecondPlusThird/src/main/java/SecondPlusThird.java; line: 20
```

Since it still does manage to build the specific exercise you're trying to run, that particular exercise is still runnable even if the full build fails. There might be a way to stop the VSCode Java plugin from trying to build everything included in the workspace, which we'll look into. In the meantime, fixing whatever issue causes those two exercises (Part03_01, Part03_02) to fail to compile should solve the issue.

**Comment by sebazai** [comment]
Hello @benjs2000 

You can deactivate autobuild for Java extension in Visual Studio Code via File -> Preferences -> Settings and search for "java.autobuild.enabled", but this means, that it won't automatically catch errors in your source code while programming, so we don't suggest this. I will close this issue, feel free to re-open or open a new one if you still encounter issues.
